docs/events/timeline-ui-overview.asciidoc

```diff
@@ -198,6 +198,8 @@ From the *Correlation* tab, you can also do the following:
 
 preview::["Do not use {esql} on production environments. This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
 
+NOTE: The {esql} tab is available by default. As it's in technical preview, you can remove it by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and setting the timelineEsqlTabDisabled feature flag value to `true`.
```

Comment:
```suggestion
NOTE: The {esql} tab is available by default. Since it's in technical preview, you can remove it by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `timelineEsqlTabDisabled: true` feature flag.
```